ddtrace/contrib/bottle/trace.py

```diff
@@ -11,17 +11,26 @@
 from ...constants import SPAN_MEASURED_KEY
 from ...ext import SpanTypes
 from ...propagation.http import HTTPPropagator
+from ...utils.formats import asbool
 
 
 class TracePlugin(object):
     name = "trace"
     api = 2
 
     def __init__(self, service="bottle", tracer=None, distributed_tracing=True):
-        self.service = ddtrace.config.service or service
+        self.service = config.service or service
         self.tracer = tracer or ddtrace.tracer
         self.distributed_tracing = distributed_tracing
 
+    @property
+    def distributed_tracing(self):
+        return config.bottle.get("distributed_tracing", True)
```

Comment:
```suggestion
        return config.bottle.distributed_tracing
```

The default is included in the config definition